frontend/src/FullPageRoutes/AccountsPage.tsx

```diff
@@ -69,7 +71,8 @@ export default function AccountsPage(): JSX.Element {
     editModeOn(!editMode)
   }
 
-  const handleSave = () => {
+  const handleSave = (e: any) => {
+    e.preventDefault().then(history.push("/"))
```

Comment:
preventDefault doesn't return a value. do we really want to redirect after a user saves?

Reply:
I agree with Eddie

Reply:
Just curious, but what does it do?

Reply:
This is an error. I don't think it was caught because the event is declared with `any` type.